content/how-to/rackspace-data-protection/index.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
rename file to `_index.md` and I think that will allow the links to start working. Looks like you are doing the correct links already.

Reply:
Rename the file as suggested _index.md